atd-vzd/migrations/default/1697144772669_add_ems_trigger_function/up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE FUNCTION public.ems_incidents_trigger()
+ RETURNS trigger
+ LANGUAGE plpgsql
+AS $function$
+BEGIN
+  update ems__incidents set
+    austin_full_purpose = (
+      select ST_Contains(jurisdiction.geometry, incidents.geometry)
+      from ems__incidents incidents
+      left join atd_jurisdictions jurisdiction on (jurisdiction.jurisdiction_label = 'AUSTIN FULL PURPOSE')
+      where incidents.id = new.id),
+    location_id = (
+      select locations.location_id
+      from ems__incidents incidents
+      join atd_txdot_locations locations on (locations.location_group = 1 and incidents.geometry && locations.shape and ST_Contains(locations.shape, incidents.geometry))
+      where incidents.id = new.id),
+    latitude = ST_Y(ems__incidents.geometry),
+    longitude = ST_X(ems__incidents.geometry),

```

Comment:
i'm sure this merely is carried forward from current production—but i'm curious why it is necessary to have these separate lat/lon columns in addition to what's stored in the geometry

Reply:
You are right on about this coming straight from prod. I'll have to look at the data source for this, but I also think that it feels denormalized!

Reply:
@patrickm02L I created https://github.com/cityofaustin/atd-data-tech/issues/14316 to capture this and the comment below.